README.md

```diff
@@ -5,7 +5,7 @@
 [![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system)
 [![Python 3.6](https://img.shields.io/badge/python-3.6-blue.svg)](https://www.python.org/downloads/release/python-360/)
 
-
+ 
```

Comment:
Can you please remove this extra space?

Reply:
@PrashanthPuneriya Done! Could you please review ?